msteams-platform/tabs/how-to/create-tab-pages/configuration-page.md

```diff
@@ -171,15 +270,31 @@ The following code provides an example of adding this function to the configurat
 ...
 ```
 
+***
+
 ## Context and authentication
 
 Authenticate before allowing a user to configure your app. Otherwise, your content might include sources that have their authentication protocols. For more information, see [authenticate a user in a Microsoft Teams tab](~/tabs/how-to/authentication/auth-flow-tab.md). Use context information to construct the authentication requests and authorization page URLs. Ensure that all domains used in your tab pages are listed in the `manifest.json` and `validDomains` array.
 
 ## Modify or remove a tab
 
-Set your manifest's `canUpdateConfiguration` property to `true`, that enables the users to modify, reconfigure, or rename a channel or group tab. Also, indicate what happens to the content when a tab is removed, by including a removal options page in the app and setting a value for the `removeUrl` property in the  `setSettings()` configuration. The user can uninstall personal tabs but cannot modify them. For more information, see [create a removal page for your tab](~/tabs/how-to/create-tab-pages/removal-page.md).
+Set your manifest's `canUpdateConfiguration` property to `true`, that enables the users to modify, reconfigure, or rename a channel or group tab. Also, indicate what happens to the content when a tab is removed, by including a removal options page in the app and setting a value for the `removeUrl` property in the `setConfig()` (formally `setSettings()`) configuration. The user can uninstall personal tabs but cannot modify them. For more information, see [create a removal page for your tab](~/tabs/how-to/create-tab-pages/removal-page.md).
 
-Microsoft Teams `setSettings()` configuration for removal page:
+Microsoft Teams `setConfig()` (formally `setSettings()`) configuration for removal page:
+
+# [TeamsJS v2](#tab/teamsjs-v2)
+
+```javascript
+app.pages.config.setConfig({
```

Comment:
pages.config.setConfig

Reply:
Addressed, along with other issues, in my most recent commit